cognite/client/_api/datapoints.py

```diff
@@ -531,6 +541,7 @@ def retrieve(
         | DatapointsQuery
         | dict[str, Any]
         | SequenceNotStr[str | DatapointsQuery | dict[str, Any]] = None,
+        instance_id: None | InstanceId | Sequence[InstanceId] = None,
```

Comment:
These are only ever node ids.
```suggestion
        instance_id: None | NodeId | Sequence[NodeId] = None,
```